### ISSUE TITLE
error

### ISSUE THREAD

**Issue by oldfart101** [woodjme/unifi-hotspot#11]
docker: Error response from daemon: driver failed programming external connectivity on endpoint elastic_burnell (bbb96e9369be6119024bb5ccfc223fdf92b2a06ecc70c88e1f0397e389a608f5): Error starting userland proxy: listen tcp 0.0.0.0:80: bind: address already in use.

**Comment by keesvv** [comment]
@oldfart101 Hi there. I've looked into the error you posted, and I noticed the following line: `listen tcp 0.0.0.0:80: bind: address already in use.`

This usually means you already have a server running at port 80, maybe a web server or an application that constantly listens on that specific port.

If you are on a Linux system, try typing `lsof -i :80` in a terminal to detect which program is listening on port 80.

If you are on a Windows system, try typing `netstat -a -b` in a Command Prompt window with Administrator privileges.
